### PR TITLE
[Bug] fix bug in time_round function

### PR DESCRIPTION
**Pull request by zhengshengjun**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
Are there any wrong cases?

**Review by dataroaring — CHANGES_REQUESTED** [review]
Please add a test case under regression-test/suites.

**Comment by BiteTheDDDDt** [comment]
Hi, you can add a test to `regression-test` or `be-unit-test`

**Comment by zhengshengjun** [comment]
@BiteTheDDDDt @dataroaring  updated

**Comment by BiteTheDDDDt** [comment]
Maybe we should fix this function on `vectorized_engine` also?
You can execute `set enable_vectorized_engine=true` to open it.
The implement is at `be/src/vec/functions/function_datetime_floor_ceil.cpp:162` .

**Comment by zhengshengjun** [comment]
> Maybe we should fix this function on `vectorized_engine` also? You can execute `set enable_vectorized_engine=true` to open it. The implement is at `be/src/vec/functions/function_datetime_floor_ceil.cpp:162` .



> Maybe we should fix this function on `vectorized_engine` also? You can execute `set enable_vectorized_engine=true` to open it. The implement is at `be/src/vec/functions/function_datetime_floor_ceil.cpp:162` .

do

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
hi @zhengshengjun you have to format the BE code style.
You can just run `sh build-support/clang-format.sh`

File: be/src/vec/functions/function_datetime_floor_ceil.cpp [hunk 1]
```diff
@@ -158,6 +158,9 @@ struct FloorCeilImpl {
     }
 };
 
+#define FLOOR 0
```
Comment (BiteTheDDDDt) [inline be/src/vec/functions/function_datetime_floor_ceil.cpp:161]: It's better to use `constexpr` to avoid `define`

Comment (zhengshengjun) [reply be/src/vec/functions/function_datetime_floor_ceil.cpp:161]: good suggestion

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
@zhengshengjun please format the BE code style

**Comment by zhengshengjun** [comment]
> @zhengshengjun please format the BE code style
Updated with correct clang-format

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**